### DISCUSSION TITLE
1st PAM Show & Tell: How'd it go?

### DISCUSSION THREAD

**Discussion by Kourtney-Burger** [nmfs-ost/PAM-OpenScience#8] [General]
Did you meet with your PAM team, how did it go? 

Did you come up with any additional pages or things to include in our lab manuals? 

How many people on your team do you anticipate will be actively contributing to your lab manual?

**Comment by jlkeating** [comment]
I have a formal group meeting set up for Nov 1st with the PAM team within our program at PIFSC. In the meantime I have met with others about what we have learned in Openscapes and it has been very helpful. There will be at least 2-3 others in our program directly contribute to the manual, but our meeting next month will include the all our PAM staff in the program. More to come.

**Comment by cberchok** [comment]
I had a quick show-n-tell at our team meeting - although a lot were on leave, so I will try to do a short demo each week.  I was able to remember how to get to the place to make edits.  I was unsure if my team can have their own discussion -where that would live - is it a part of the lab manual or some other place. Would be good to start building the habit of going there instead of using emails.  Same with the project management app.   Everyone is on board with the lab manual and I will be pulling various people in to help with certain sections.  One comment was that it would be great to have a very clear table of contents (which is already a part of the manual) so that you don't have to randomly guess where a piece of information is.  Probably 6 people will be contributing in some way or another.
